### PR TITLE
adds infura information to initial-state

### PR DESCRIPTION
**Pull request by cooganb**
## Pull Request Description

Hi there! This adds the information about Infura's ETH2 offering of an initial state to its users.

### CODE REVIEW

**Comment by cooganb** [comment]
@NicolasMassart Sounds good! I believe I merged your suggestion

**Review by bgravenorst — APPROVED** [review]
Looks good to me.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: docs/HowTo/Get-Started/Checkpoint-Start.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ on the beacon chain client (for example Teku).
     ```bash
     curl -o state.ssz -H 'Accept: application/octet-stream' http://other-node:5051/eth/v1/debug/beacon/states/finalized
     ```
+    [Infura](https://infura.io/) provides initial state endpoints for free.
+    Registration to Infura services is required.
+    Watch the ("Teku Snapshot Sync from Infura" video)[https://youtu.be/ce9uVRl23zI] for more details.
```
Comment (ajsutton) [inline docs/HowTo/Get-Started/Checkpoint-Start.md:42]: Actually as per other discussions:
```suggestion
    Watch the ("Teku Checkpoint Sync from Infura" video)[https://youtu.be/ce9uVRl23zI] for more details.
```

**Comment by ConsenSys-Academy** [comment]
Wanted to see what the status of merging this is? @ajsutton 's changes are good for me

**Comment by ConsenSys-Academy** [comment]
Also, we may want to change the title of the doc to reflect the consistent terminology, "Checkpoint Sync"

**Review by NicolasMassart — APPROVED** [review]
Looks good to me.

**Pull request merged**